tests/src/test/java/alluxio/hadoop/HdfsFileInputStreamIntegrationTest.java

```diff
@@ -53,25 +59,29 @@
 
   @BeforeClass
   public static final void beforeClass() throws Exception {
-    FileSystem fileSystem = sLocalAlluxioClusterResource.get().getClient();
-    FileSystemTestUtils
-        .createByteFile(fileSystem, "/testFile1", WriteType.CACHE_THROUGH, FILE_LEN);
-    FileSystemTestUtils.createByteFile(fileSystem, "/testFile2", WriteType.THROUGH, FILE_LEN);
+    sFileSystem = sLocalAlluxioClusterResource.get().getClient();
   }
 
   @After
   public final void after() throws Exception {
     mInMemInputStream.close();
     mUfsInputStream.close();
+    sFileSystem.delete(new AlluxioURI(IN_MEMORY_FILE));
+    sFileSystem.delete(new AlluxioURI(UFS_ONLY_FILE));
   }
 
   @Before
   public final void before() throws IOException, AlluxioException {
-    AlluxioURI file1 = new AlluxioURI("/testFile1");
-    mInMemInputStream = new HdfsFileInputStream(file1, new Configuration(), BUFFER_SIZE, null);
+    FileSystemTestUtils
+        .createByteFile(sFileSystem, IN_MEMORY_FILE, WriteType.CACHE_THROUGH, FILE_LEN);
+    FileSystemTestUtils.createByteFile(sFileSystem, UFS_ONLY_FILE, WriteType.THROUGH, FILE_LEN);
+
+    mInMemInputStream = new HdfsFileInputStream(
+        new AlluxioURI(IN_MEMORY_FILE), new Configuration(), BUFFER_SIZE, null);
 
-    AlluxioURI file2 = new AlluxioURI("/testFile2");
-    mUfsInputStream = new HdfsFileInputStream(file2, new Configuration(), BUFFER_SIZE, null);
+    ClientContext.getConf().set(Constants.USER_FILE_READ_TYPE_DEFAULT, "NO_CACHE");
```

Comment:
I see - in that case could we do something similar to the start of readCacheTest in the individual tests which require the read type to be NO_CACHE? I'd rather avoid setting global configuration state in `before` if possible.